examples/ITR_UI.py

```diff
@@ -119,6 +124,12 @@ def recalculate_individual_itr(scenario):
         parsed_json = json.load(json_file)
     EI_bm = BaseProviderIntensityBenchmark(EI_benchmarks=IEIBenchmarkScopes.parse_obj(parsed_json))
     Warehouse = DataWarehouse(template_company_data, base_production_bm, EI_bm)
+    temperature_score = TemperatureScore(
+                            time_frames = [ETimeFrames.LONG],
+                            scopes=[EI_bm.scope_to_calc],
```

Comment:
i was re-running this branch after "S3-only" merge and rebase, and `ITR_UI.py` fails for me in this line, with exception:

~~~~
2022-12-04 20:34:47,679 - __main__ - INFO - Load dummy portfolio from /home/kirill/os-climate/ITR/examples/data/20220927 ITR V2 Sample Data.xlsx. You could upload your own portfolio using the template.
Traceback (most recent call last):
  File "/home/kirill/os-climate/ITR/examples/ITR_UI.py", line 137, in <module>
    initial_portfolio = recalculate_individual_itr('OECM_PC')
  File "/home/kirill/os-climate/ITR/examples/ITR_UI.py", line 129, in recalculate_individual_itr
    scopes=[EI_bm.scope_to_calc],
AttributeError: 'BaseProviderIntensityBenchmark' object has no attribute 'scope_to_calc'
~~~~

does it work for you?

Reply:
Clearly a merge error.  I'll take a look.  I did do some testing with ITR_UI.py, but probably forgot to test as I focused on unit tests.  Any thoughts on how we could include ITR_UI in the unit testing?

Reply:
I don't think we need to include ITR_UI to unit tests.
The core functionality is not yet covered until a degree i consider comfortable - let's focus on them, first

Reply:
OK, so I've taken a look at how this crept in, and also why it was not caught in the unit tests.  And it does come down to some fairly important design questions about whether or how we want scope decisions to be entirely automated by benchmarks and data, or whether we want to try to support some concept of users asking questions that can be answered by granular scope data.  In any case, I took this field out of the benchmark because in the case of TPI benchmarks, the relevant scope or combination of scopes is sector-dependent, and there really isn't a concept of a single answer of `scope_to_calc` for all our benchmarks.  But we do need to run this problem down to a satisfactory set of functionality and implementation decisions.

Reply:
which short-term solution would you see reasonable, to fix behaviour of UTR_UI as a part of this PR?
we can always do a more thought-through "how we want scope decisions to be entirely automated" design decision later, in a separate work package